crates/uv/tests/venv.rs

```diff
@@ -149,18 +51,18 @@ fn create_venv() {
 
 #[test]
 fn create_venv_defaults_to_cwd() {
-    let context = VenvTestContext::new(&["3.12"]);
-    uv_snapshot!(context.filters(), context.venv_command()
+    let context = TestContext::new_with_versions(&["3.12"]);
+    uv_snapshot!(context.filters(), context.venv()
         .arg("--python")
         .arg("3.12"), @r###"
     success: true
     exit_code: 0
     ----- stdout -----
 
     ----- stderr -----
-    Using Python 3.12.[X] interpreter at: [PATH]
-    Creating virtualenv at: .venv
-    Activate with: source .venv/bin/activate
+    Using Python 3.12.[X] interpreter at: [PYTHON-3.12]
+    Creating virtualenv at: [VENV]/
+    Activate with: source [VENV]/bin/activate
```

Comment:
Can we keep them as `.venv`? These tests are meant to check that we create the venv in the right place with the right name

Reply:
I'll check.

Reply:
So the problem here is we actually have different displays on each platform when you pass the full path like we do in these tests. I'll see what I can do.